packit_service/worker/checker/distgit.py

```diff
@@ -139,17 +120,7 @@ def pre_check(self) -> bool:
                 )
                 return False
 
-            if self.data.event_dict["committer"] == "pagure":
-                if not self.pull_request:
-                    logger.debug(
-                        "Not able to get the pull request "
-                        "(may not be the head commit of the PR)."
-                    )
-                    return False
-
-                if not self.contains_specfile_change():
-                    return False
-
+            if self.pull_request:
```

Comment:
@nforro as you mentioned, there is no way to tell from the message whether the commit(s) is coming from PR, do you think it is ok to always try to get the PR via `end_commit` or do you have some other suggestions?

Reply:
I don't think we have a choice. But I think this has to be adjusted as well:

https://github.com/packit/packit-service/blob/a439cb5d734252ae00cb1f8143cbba301def7571/packit_service/worker/mixin.py#L257

Reply:
> I don't think we have a choice. 

there is still option to revisit the 2 config options, but that would require changes in configs for users who use these options

Reply:
I was wondering if it still makes sense to differentiate between the two events. But we need to be able to recognize Packit PRs, and the person making the push (agent) is not packit, e.g.: https://apps.fedoraproject.org/datagrepper/v2/id?id=a934e891-c91c-4d7e-b287-3d7ef6771f8f&is_raw=true&size=extra-large.

Reply:
Yes I know, therefore I wrote it would require changes in the configs (I meant having only one option, for the committer, but the default could not be `packit` and would be more complicated). I will go with the current implementation and if we notice some problems with this, we can revisit.